.flake8

```diff
@@ -0,0 +1,20 @@
+[flake8]
+extend-ignore =
+    # existing errors
+    E30,E501,
+    E124,E26,E401,E402,E731
+    # flake8-future-import "__future__ import present"
```

Comment:
What is `present`? (I don't see it in the docs)

Reply:
That is part of the error, not code, I've adjusted it to include a placeholder.